commands/displayers/droplet_backup_policies.go

```diff
@@ -0,0 +1,50 @@
+package displayers
+
+import (
+	"io"
+
+	"github.com/digitalocean/doctl/do"
+)
+
+type DropletBackupPolicy struct {
+	DropletBackupPolicies []do.DropletBackupPolicy
+}
+
+var _ Displayable = &DropletBackupPolicy{}
+
+func (d *DropletBackupPolicy) JSON(out io.Writer) error {
+	return writeJSON(d.DropletBackupPolicies, out)
+}
+
+func (d *DropletBackupPolicy) Cols() []string {
+	cols := []string{
+		"DropletID", "BackupEnabled", "BackupPolicyPlan", "BackupPolicyWeekday", "BackupPolicyHour",
+		"BackupPolicyWindowLengthHours", "BackupPolicyRetentionPeriodDays",
+		"NextBackupWindowStart", "NextBackupWindowEnd",
+	}
+	return cols
+}
+
+func (d *DropletBackupPolicy) ColMap() map[string]string {
+	return map[string]string{
+		"DropletID": "Droplet ID", "BackupEnabled": "Backup Enabled",
+		"BackupPolicyPlan": "Backup Policy Plan", "BackupPolicyWeekday": "Backup Policy Weekday", "BackupPolicyHour": "Backup Policy Hour",
+		"BackupPolicyWindowLengthHours": "Backup Policy Window Length Hours", "BackupPolicyRetentionPeriodDays": "Backup Policy Retention Period Days",
+		"NextBackupWindowStart": "Next Backup Window Start", "NextBackupWindowEnd": "Next Backup Window End",
+	}
+}
+
+func (d *DropletBackupPolicy) KV() []map[string]any {
+	out := make([]map[string]any, 0)
+	for _, d := range d.DropletBackupPolicies {
+		m := map[string]any{
+			"DropletID": d.DropletID, "BackupEnabled": d.BackupEnabled, "BackupPolicyPlan": d.BackupPolicy.Plan,
+			"BackupPolicyWeekday": d.BackupPolicy.Weekday, "BackupPolicyHour": d.BackupPolicy.Hour,
+			"BackupPolicyWindowLengthHours": d.BackupPolicy.WindowLengthHours, "BackupPolicyRetentionPeriodDays": d.BackupPolicy.RetentionPeriodDays,
+			"NextBackupWindowStart": d.NextBackupWindow.Start, "NextBackupWindowEnd": d.NextBackupWindow.End,
+		}
```

Comment:
We need to protect against panics here for Droplets that do not have backups enabled:

```
$ ./builds/doctl compute droplet list-backup-policies
si:  [0xc00009a280 0xc00009a320 0xc00009a400 0xc00009a2c0 0xc00009a360 0xc00009a3a0 0xc00009a480 0xc00009a300 0xc00009a340 0xc00009a440 0xc00009a460 0xc00009a2e0 0xc00009a380 0xc00009a3c0 0xc00009a3e0 0xc00009a420]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa74b84]

goroutine 1 [running]:
github.com/digitalocean/doctl/commands/displayers.(*DropletBackupPolicy).KV(0x1364b40?)
        github.com/digitalocean/doctl/commands/displayers/droplet_backup_policies.go:41 +0x184
github.com/digitalocean/doctl/commands/displayers.DisplayText({0x1371278, 0xc00056a078}, {0x13644a0, 0xc00007a070}, 0x0, {0x0, 0x0, 0x2?})
        github.com/digitalocean/doctl/commands/displayers/output.go:91 +0x33b
github.com/digitalocean/doctl/commands/displayers.(*Displayer).Display(0xc000113bd0)
        github.com/digitalocean/doctl/commands/displayers/output.go:61 +0x1e8
github.com/digitalocean/doctl/commands.(*CmdConfig).Display(0xc000364340, {0x1371278?, 0xc00056a078?})
        github.com/digitalocean/doctl/commands/command_config.go:226 +0x115
github.com/digitalocean/doctl/commands.RunDropletListBackupPolicies(0xc000364340)
        github.com/digitalocean/doctl/commands/droplets.go:879 +0x95
github.com/digitalocean/doctl/commands.cmdBuilderWithInit.func1(0xc000362900?, {0x1b78300, 0x0, 0x0})
        github.com/digitalocean/doctl/commands/command.go:103 +0xe4
github.com/spf13/cobra.(*Command).execute(0xc0001a7808, {0x1b78300, 0x0, 0x0})
        github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x1b3f040)
        github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.7.0/command.go:992
github.com/digitalocean/doctl/commands.Execute()
        github.com/digitalocean/doctl/commands/doit.go:161 +0x25
main.main()
        github.com/digitalocean/doctl/cmd/doctl/main.go:24 +0x4a
```

Reply:
Thanks for catching this. Updated in [b796bb5](https://github.com/digitalocean/doctl/pull/1609/commits/b796bb581a1a3a6ed97f07ae038beccaebce3abc)